hawc/apps/lit/templates/lit/ris_export_instructions.html

```diff
@@ -55,13 +55,13 @@ <h2 id="importing-data-into-endnote">Importing data into EndNote</h2>
 
 
 
-<h3 id="pubmed">PubMed:</h3>
+<h4 id="pubmed">PubMed:</h4>
 
 <p>For PubMed import into EndNote, use the Medline format and save the text file. Import using PubMed filter customized for HAWC imports (<kbd>HAWC_PubMed</kbd>). “NLM” will appear in Database Name field.</p>
 
 
 
-<h4 id="copying-the-pmid-into-the-custom-8-field">Copying the PMID into the Custom 8 field</h4>
+<h5 id="copying-the-pmid-into-the-custom-8-field">Copying the PMID into the Custom 8 field</h5>
```

Comment:
Ended up using a couple <h5> tags for this one page

Reply:
makes sense, the nesting here is probably too deep